src/App.tsx

```diff
@@ -1,24 +1,82 @@
 import React from 'react';
 import './App.scss';
+import { Clock } from './Components/Clock/Clock';
 
-const App: React.FC = () => {
-  const timerId: NodeJS.Timer = setInterval(() => {
-    const date: Date = new Date();
-
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
+type Props = {};
+
+type State = {
+  isClockVisible: boolean;
+  clockName: number;
 };
 
+class App extends React.Component<Props, State> {
+  state = {
+    isClockVisible: true,
+    clockName: Math.floor(Math.random() * 100),
+  };
+
+  visibleClock = () => {
```

Comment:
Function name should describe an action so there should be a verb in its name
```suggestion
  makeClockVisible = () => {
```